integration/service-definitions/simple.ts

```diff
@@ -0,0 +1,293 @@
+/* eslint-disable */
+import { util, configure, Writer, Reader } from 'protobufjs/minimal';
+import * as Long from 'long';
+import { Timestamp } from './google/protobuf/timestamp';
+import { Empty } from './google/protobuf/empty';
+import {
+  StringValue,
+  Int64Value,
+  UInt64Value,
+  Int32Value,
+  UInt32Value,
+  BytesValue,
+  FloatValue,
+  DoubleValue,
+  BoolValue,
+} from './google/protobuf/wrappers';
+
+export const protobufPackage = 'simple';
+
+export interface TestMessage {
+  timestamp: Date | undefined;
+}
+
+const baseTestMessage: object = {};
+
+export const TestMessage = {
+  encode(message: TestMessage, writer: Writer = Writer.create()): Writer {
+    if (message.timestamp !== undefined) {
+      Timestamp.encode(toTimestamp(message.timestamp), writer.uint32(10).fork()).ldelim();
+    }
+    return writer;
+  },
+
+  decode(input: Reader | Uint8Array, length?: number): TestMessage {
+    const reader = input instanceof Reader ? input : new Reader(input);
+    let end = length === undefined ? reader.len : reader.pos + length;
+    const message = { ...baseTestMessage } as TestMessage;
+    while (reader.pos < end) {
+      const tag = reader.uint32();
+      switch (tag >>> 3) {
+        case 1:
+          message.timestamp = fromTimestamp(Timestamp.decode(reader, reader.uint32()));
+          break;
+        default:
+          reader.skipType(tag & 7);
+          break;
+      }
+    }
+    return message;
+  },
+
+  fromJSON(object: any): TestMessage {
+    const message = { ...baseTestMessage } as TestMessage;
+    if (object.timestamp !== undefined && object.timestamp !== null) {
+      message.timestamp = fromJsonTimestamp(object.timestamp);
+    } else {
+      message.timestamp = undefined;
+    }
+    return message;
+  },
+
+  toJSON(message: TestMessage): unknown {
+    const obj: any = {};
+    message.timestamp !== undefined && (obj.timestamp = message.timestamp.toISOString());
+    return obj;
+  },
+
+  fromPartial(object: DeepPartial<TestMessage>): TestMessage {
+    const message = { ...baseTestMessage } as TestMessage;
+    if (object.timestamp !== undefined && object.timestamp !== null) {
+      message.timestamp = object.timestamp;
+    } else {
+      message.timestamp = undefined;
+    }
+    return message;
+  },
+};
+
+/**
+ * Test
+ *
+ * @deprecated
+ */
+export const TestDefinition = {
```

Comment:
I've added the `Definition` suffix as per [discussion](https://github.com/stephenh/ts-proto/pull/262#discussion_r606353216).

Though now it seems that the suffix should be `ServiceDefinition`

Reply:
Hey @stephenh, thank you for review. What do you think about this one?

Reply:
After some thinking, I'm fine with `Definition` suffix as long as we don't overload this term and only use it for services.